### PR TITLE
[Blog] week 11

### PR DESCRIPTION
**Pull request by lbarcziova**

### CODE REVIEW

File: content/posts/March-2020.md [hunk 1]
```diff
@@ -38,3 +38,31 @@ draft: false
...
+    * We handle the `git-describe` output better which should help when tags contain dashes [(#759)]
+* Packit does not add the patches inside the macro now [(#760)]
+
+
+### packit-service
+* Comments from packit-service about successful `propose-update` in the issue which triggered the update now include the URL of 
+  the new pull-request in Fedora [(#472)]
+* We have improved the mapping of handlers to events [(#476)]
+* The JobTriggerType was split to one related to config and one for actual 
+  triggers [(#746)], [(#453)]
```
Comment (TomasTomecek) [inline content/posts/March-2020.md]: I'd comment these as: "we have done some major refactoring in packit-service how jobs are processed"

File: content/posts/March-2020.md [hunk 2]
```diff
@@ -38,3 +38,31 @@ draft: false
...
+### packit
+* Log messages when loading a package config are improved to be more clear [(#755)]
+* We have changed how packit treats version and release in the spec file [(#748)]
+    * We handle the `git-describe` output better which should help when tags contain dashes [(#759)]
+* Packit does not add the patches inside the macro now [(#760)]
+
+
+### packit-service
+* Comments from packit-service about successful `propose-update` in the issue which triggered the update now include the URL of 
+  the new pull-request in Fedora [(#472)]
```
Comment (TomasTomecek) [inline content/posts/March-2020.md]: had to read this one like 3 times - what a complex sentence :D let's leave it as it is

File: content/posts/March-2020.md [hunk 3]
```diff
@@ -38,3 +38,31 @@ draft: false
 [(#458)]: https://github.com/packit-service/packit-service/pull/458
 [(#455)]: https://github.com/packit-service/packit-service/pull/455
 
+
+## Week 11 (March 9th - March 13th)
+
+
+
+### packit
+* Log messages when loading a package config are improved to be more clear [(#755)]
```
Comment (TomasTomecek) [inline content/posts/March-2020.md]: I'd remove this one since users are not exposed to this.

**Comment by TomasTomecek** [comment]
I'd also add these:
```
* Rename packit generate to packit init, by [@shreyaspapi](https://github.com/shreyaspapi), [#728](https://github.com/packit-service/packit/pulls/728)
￼  * description: '#701 I have done the changes, please check if this is what you meant.'
￼  * commit: Renaming generate to init with aliasing

Easier local development of p-s, by [@TomasTomecek](https://github.com/TomasTomecek), [#473](https://github.com/packit-service/packit-service/pulls/473)
￼  * description: '* make debugging of p-s easier\n* document how to run p-s httpd locally\n* put httpd config for p-s in this repo'
￼  * commit: make debugging of p-s easier
￼  * commit: document how to run p-s httpd locally
￼  * commit: put httpd config for p-s in this repo
```

And probably even your reverse testing change :)

**Comment by lbarcziova** [comment]
> I'd also add these:
> 
> ```
> * Rename packit generate to packit init, by [@shreyaspapi](https://github.com/shreyaspapi), [#728](https://github.com/packit-service/packit/pulls/728)
> ￼  * description: '#701 I have done the changes, please check if this is what you meant.'
> ￼  * commit: Renaming generate to init with aliasing
This is included in the week 10, isn't it?

> Easier local development of p-s, by [@TomasTomecek](https://github.com/TomasTomecek), [#473](https://github.com/packit-service/packit-service/pulls/473)
> ￼  * description: '* make debugging of p-s easier\n* document how to run p-s httpd locally\n* put httpd config for p-s in this repo'
> ￼  * commit: make debugging of p-s easier
> ￼  * commit: document how to run p-s httpd locally
> ￼  * commit: put httpd config for p-s in this repo
> ```
> 
> And probably even your reverse testing change :)

OK, I will add them, I was just not sure if they are interesting enought for the users

File: content/posts/March-2020.md [hunk 4]
```diff
@@ -38,3 +38,35 @@ draft: false
...
+    contain dashes [(#759)]
+* Packit does not add the patches inside the macro now [(#760)]
+* In pull request pipelines we run the packit-service tests so that we are sure we 
+  do not break the service [(#752)]
+
+
+### packit-service
+* Comments from packit-service about successful `propose-update` in the issue 
+  which triggered the update now include the URL of the new pull-request in Fedora [(#472)]
+* We have done some major refactoring in packit-service how jobs are processed
```
Comment (jpopelka) [inline content/posts/March-2020.md]: This is already in `packit-service` section

```suggestion
* We have done some major refactoring in how jobs are processed 
```

File: content/posts/March-2020.md [hunk 5]
```diff
@@ -38,3 +38,35 @@ draft: false
...
+
+## Week 11 (March 9th - March 13th)
+
+
+
+### packit
+* We have changed how packit treats version and release in the spec file [(#748)]
+    * We handle the `git-describe` output better which should help when tags 
+    contain dashes [(#759)]
+* Packit does not add the patches inside the macro now [(#760)]
```
Comment (jpopelka) [inline content/posts/March-2020.md]: This needs a bit more context:
```suggestion
* When packit adds source-git patches into a spec file it adds them outside of rpm macros now [(#760)]
```

Comment (lbarcziova) [reply content/posts/March-2020.md]: thanks, fixed

**Pull request merged**